packages/core/src/queries/organization/index.ts

```diff
@@ -116,6 +116,33 @@ class OrganizationInvitationsQueries extends SchemaQueries<
     return this.pool.any(this.#findEntity({ ...options, invitationId: undefined }));
   }
 
+  async updateExpiredEntities({
+    invitationId,
+    organizationId,
+    inviterId,
+    invitee,
+  }: OrganizationInvitationSearchOptions): Promise<void> {
+    const { table, fields } = convertToIdentifiers(OrganizationInvitations);
+    await this.pool.query(sql`
+      update ${table}
+      set ${fields.status} = ${OrganizationInvitationStatus.Expired}
+      where ${fields.status} = ${OrganizationInvitationStatus.Pending}
+      and ${fields.expiresAt} < now()
+      ${conditionalSql(invitationId, (id) => {
+        return sql`and ${fields.id} = ${id}`;
+      })}
```

Comment:
do we need these conditional checks? i couldn't see any obvious reason for them

Reply:
Removed